app/build.gradle.kts

```diff
@@ -217,6 +222,9 @@ dependencies {
 
     // Library/extensions searching with Levenshtein distance
     implementation("me.xdrop:fuzzywuzzy:1.4.0")
+    implementation("com.github.holgerbrandl:themoviedbapi:1.12") {
```

Comment:
The issue lies with using another API, not that the code is in the wrong file

Reply:
You never want unnecessary dependencies

Reply:
See https://github.com/recloudstream/cloudstream/blob/master/app/build.gradle.kts#L197
We already have a dependency to handle the JSON that the API returns.

Reply:
But that method uses com.github.holgerbrandl:themoviedbapi, the unnecessary dependency

Reply:
Because the other API already has a sizable codebase in the app which works, not to mention https://github.com/UweTrottmann/tmdb-java has more recent commits and releases